tests/foreman/maintain/test_health.py

```diff
@@ -760,7 +767,7 @@ def test_positive_health_check_non_rh_packages(sat_maintain, request):
         == 0
     )
     result = sat_maintain.cli.Health.check({'label': 'non-rh-packages'})
-    assert 'Found 1 unexpected non Red Hat Package(s) installed!' in result.stdout
+    assert 'unexpected non Red Hat Package(s) installed!' in result.stdout
```

Comment:
I remember this, here this f-m procedure also puts count in error if non-rh packages installed, so I think it'll be nice to have it tested, and if instead of whitelist above if we remove it, then it wouldn't be impacted, wdyt? non-blocking

Reply:
@Gauravtalreja1 Out of curiosity, what was the one non-RH-signed package?

Reply:
@ogajduse we use a package from our fedorapeople container to test this non-rh signed package